modular_skyrat/modules/derelictdrone/code/space.dm

```diff
@@ -0,0 +1,6 @@
+/datum/map_template/ruin/space/dronesderelict
```

Comment:
Please separate the words with underscores in typepaths (and in your folder hierarchy too, please <3)
```suggestion
/datum/map_template/ruin/space/drones_derelict
```